src/index.js

```diff
@@ -1160,7 +1236,7 @@ follow these steps, YOUR APP MAY BREAK.`);
  * @private
  * @type {Firestore~logFunction}
  */
-Firestore.log = function() {};
+Firestore.log = function(methodName, logMessage, varargs) {};
```

Comment:
Should this be `methodName, requestTag, varargs` for consistency?

Reply:
Hm, I do want to make it `methodName, requestTag, varargs`, but it turns out that the linter complains since these arguments are unused. Back to `() => {}` it is.